website/registries/views.py

```diff
@@ -19,13 +19,14 @@ def _view_registries_landing_page(campaign=None, **kwargs):
     else:
         has_projects = False
 
+    campaign_url_param = 'osf-registered-reports' if campaign == 'registered_report' else 'prereg'
```

Comment:
If these get changed to ember-osf-web pages, I expect this won't continue to work. Do you know how the other ember-osf-web pages get the campaign? I feel like @jamescdavis may have said it comes from the back-end, but I'm not positive.

So, if it does come from the back-end, can you see if there's a way to pre-fix that so that, if the pages are embosfed, this continues to work? Alternately, if it's all front-end work, can you see if Product can set a requirement that those pages include this campaign?

Reply:
Currently, it seems that the only embosfed pages where we have campaign query params included in the navbar sign up link are on Registries. We manually pass a `@campaign='osf-registries'` param to the navbar component. And the navbar component would construct the query params. See [here](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/lib/registries/addon/application/template.hbs#L4).

Reply:
Yeah, that seems fine. [This](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/lib/osf-components/addon/components/sign-up-form/component.ts#L104-L106) has a parameter for campaign that does the rest, which I believe is what ultimately receives that. I'll let Product know to add a requirement in for that.